### ISSUE TITLE
Return project on successful fork

### ISSUE THREAD

**Issue by Spenhouet** [gmessner/gitlab4j-api#106]
In the current implementation nothing is returned from the `forkProject` method.

The GitLab API return the new project (the forked project).
It would be good if the Java API would return a `Project` instance of the forked project (the returned project by the GitLab API).

**Comment by gmessner** [comment]
Unfortunately, the forking operation for a project is asynchronous and is completed in a background job. The request will return immediately.  You'll need to enter a loop and check for the project with:
```java
public Project getProject(String namespace, String project)
```
Until it returns successfully.

**Comment by Spenhouet** [comment]
The API call for fork actually returns the forked project.

That should be enough, or doesn't it?

**Comment by gmessner** [comment]
The docs do not indicate that a Project instance is returned, however, that would be unusual.  I have went ahead and added returning the Project, but have not yet added a test case for same.  I'll get to this later today and do another release if it works.

**Comment by Spenhouet** [comment]
As far as I understand the documentation, the git fork operation is done in the background but the GitLab project is created directly. To check if the git fork operation finished the documentation recommends to check the projects `import_status`.

As I see it, the `Project` object only contains the project information anyway, so returning a `Project` instance shouldn't be a problem. If someone wants to know if the actual fork is finished he could still loop to check for the `import_status`.

Thanks for taking a second look at it 👍 I hope I'm right with my assessment.

**Comment by gmessner** [comment]
Returning the newly forked Project instance has been added and tested.  Is available in release 4.7.3.
